_posts/2017-09-04-Secure-your-architecture-part1.md

```diff
@@ -0,0 +1,422 @@
+---
+layout: post
+authors: [kevin_van_houtte]
+title: 'Securing your cloud-native microservice architecture in Spring: part 1'
+image: /img/microservices/part1/securitylogo.png
+tags: [Microservices, Security, OAuth2, JWT, Redis, Session, Spring]
+category: Microservices
+comments: true
+---
+
+When developing cloud-native microservices, we need to think about securing the data that is being propagated from one service to another service and securing the data at rest. 
+So how much security is enough to secure our architecture? Is it the user that identifies itself and decides what data he has access to?
+In this blog series I will cover these questions and guide you in applying the security layer to your cloud-native blueprint.
```

Comment:
It typically is better to use "we" instead of "I".
That way whole of JWorks is backing this. :-)